### DISCUSSION TITLE
https://seul1230.github.io/daily/sseul-daily-20240613

### DISCUSSION THREAD

**Discussion by giscus** [seul1230/giscus#6] [Comments]
# https://seul1230.github.io/daily/sseul-daily-20240613

sseul's blog

https://seul1230.github.io/daily/sseul-daily-20240613

**Comment by J-PARK11** [comment]


**Reply by seul1230** [reply]
아유 감사합니다 :D 
조금씩 더 올려볼게요 ~ !

**Comment by eoslovy** [comment]
여행 계획을 보니 mbti가 j 같은데 맛집 웨이팅 몇 분까지 기다릴 수 있나요? 혹시 시간 단위인가요?

**Reply by seul1230** [reply]
ㅋㅋㅋㅋㅋ감사합니다. MBTI는 ISTP입니다ㅎ.ㅎ 
맛집 웨이팅은 2시간까지 기다릴 수 있어요 현장 웨이팅은,,, 1시간? 😇
